modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1788,8 +1778,8 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return response;
   }
 
-  protected abstract getAddressInfoFromExplorer(address: string, apiKey?: string): Bluebird<AddressInfo>;
-  protected abstract getUnspentInfoFromExplorer(address: string, apiKey?: string): Bluebird<UnspentInfo[]>;
+  public abstract getAddressInfoFromExplorer(address: string, apiKey?: string): Bluebird<AddressInfo>;
+  public abstract getUnspentInfoFromExplorer(address: string, apiKey?: string): Bluebird<UnspentInfo[]>;
```

Comment:
I'm not sure we want to add this to the public api, since it's really supposed to only be used for recoveries. Instead, maybe keep these `protected` and pass the functions into `recover`?

Reply:
if it really is only used for recoveries IMO we should move it to `/recover.ts` and use `switch` instead of subclassing